docs/src/components/VersionsTable.tsx

```diff
@@ -0,0 +1,201 @@
+import {
+  ResponsiveTable,
+  ResponsiveTableCell,
+} from '@/components/ResponsiveTable';
+import {
+  Tabs,
+  TableCell,
+  TableBody,
+  TableHead,
+  TableRow,
+} from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+import {
+  Framework,
+  PREV_MAJOR_VERSIONS,
+  CURRENT_MAJOR_VERSIONS,
+} from '../data/frameworks';
+
+export const VersionsTable = ({ framework }) => {
+  const {
+    query: { platform = 'react' },
+  } = useRouter();
+
+  if (!framework) {
+    framework = platform as Framework;
+  }
+
+  const latestVersion = CURRENT_MAJOR_VERSIONS[framework].toString();
+  const prevVersion = PREV_MAJOR_VERSIONS[framework].toString();
+
+  return (
+    <Tabs.Container defaultValue={latestVersion}>
+      <Tabs.List>
+        <Tabs.Item value={latestVersion}>
+          ui-{framework} v{latestVersion} (latest)
+        </Tabs.Item>
+        <Tabs.Item value={prevVersion}>
+          ui-{framework} v{prevVersion}
+        </Tabs.Item>
+      </Tabs.List>
+      <Tabs.Panel value={latestVersion}>
+        <ResponsiveTable labelWidth="10rem">
```

Comment:
How does this look on mobile?

Reply:
It looks normal/same as the other tables! I didn't change this from the previous setup just copied over into a new file